LLK/scratch-www
# converted images around splash video to png, smaller file size

### Resolves:

Partially addresses https://github.com/LLK/scratch-www/issues/5113

### Changes:

Reduces the size of one of the splash page's images, from 33k->13k, and 41k->17k.

### Testing plan:

* Make sure the images around the video on the splash page appear, and look good.
* Try both with wide window and narrow window

### Before:

![image](https://user-images.githubusercontent.com/3431616/110494986-163c6680-80c2-11eb-89eb-a56b4df36330.png)

### After:

(need image)
